docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -440,6 +441,18 @@ This key can be listed multiple times.
 The (numeric) GID to run as inside the container. This does not need to match the GID on the host,
 which can be modified with `UsersNS`, but if that is not specified, this GID is also used on the host.
 
+### `GroupAdd=`
```

Comment:
To avoid duplication, the Quadlet man page usually does not provide the entire documentation of the field. Instead, it states the equivalent argument.
So, I guess here you can keep the first sentence, maybe also the note about `keep-groups` and conclude with "Equivalent to the Podman `--group-add` option"